src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/ConsignmentFields.scala

```diff
@@ -134,7 +136,11 @@ object ConsignmentFields {
       Field(
         "files",
         ListType(FileType),
-        resolve = context => DeferFiles(context.value.consignmentid)
+        arguments = FileFiltersInputArg :: Nil,
```

Comment:
The `fileFiltersInputArg` will need to be passed into the `getConsignment` query (line 188) as an optional argument I think, otherwise it won't be accessible by the deferred resolve for getting the files.
...
```
Field("getConsignment", OptionType(ConsignmentType),
      arguments = ConsignmentIdArg :: PaginationInputArg :: FileFiltersInputArg :: Nil,
      ...
    )
...
```

Reply:
Actually, I tested and it works fine. I think we don't need to pass with `getConsignment` 2field as we are using the `fileFiltersInputArg` in the `files` field only

Reply:
As per our conversation, yes correct. Please ignore this comment.